### PR TITLE
feat: create Workspace Mapping

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

https://github.com/cloudforet-io/console/assets/50662170/792f3556-5b45-449f-bcea-d33b06a23f40

### CODE REVIEW

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue [hunk 1]
```diff
@@ -59,7 +60,7 @@ interface SelectDropdownProps {
     handler?: AutocompleteHandler;
     disableHandler?: boolean;
     pageSize?: number;
-    resetSelectedOnUnmounted?: boolean;
+    resetSelectedOnUnmounted?: boolean;s
```
Comment (yuda110) [inline packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:63]: quibble: 😲???

File: apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue [hunk 2]
```diff
@@ -37,11 +39,88 @@ const emit = defineEmits<{(e:'update:isValid', isValid: boolean): void;
     (e:'change', formData: any): void;
 }>();
 
+const cspAdditionalOptionMap = {
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue:42]: question: Is this sample? or real data??

File: apps/web/src/services/asset-inventory/components/WorkspaceDropdown.vue [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+        state.loading = false;
+    }
+};
+const handleClickReloadButton = async () => {
+    state.pageStart = 1;
+    state.workspaceList = await fetchWorkspace();
+};
+
+const handleClickCreateButton = () => {
+    window.open(router.resolve({ name: makeAdminRouteName(PREFERENCE_ROUTE.WORKSPACES._NAME) }).href);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/components/WorkspaceDropdown.vue:89]: suggestion: It'd be better to add `_blank` option!

```typescript
window.open(router.resolve({ name: makeAdminRouteName(PREFERENCE_ROUTE.WORKSPACES._NAME) }).href, '_blank');
```

File: apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue [hunk 4]
```diff
@@ -108,64 +108,71 @@ const handleAdditionalOptionsValidate = (isValid) => {
     <div class="service-account-auto-sync-form">
         <div class="auto-sync-toggle">
             <p-toggle-button v-if="serviceAccountPageStore.getters.autoSyncAdditionalOptions"
-                             v-model="state.isAutoSyncEnabled"
+                             :value="state.isAutoSyncEnabled"
                              show-state-text
                              position="left"
+                             @change-toggle="state.isAutoSyncEnabled = $event"
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue:209]: Suggestion: using `handleChangeToggle` function is better~

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue (hunk 1 above)
Comment (piggggggggy) [reply packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:63]: ㅋㅋㅋ

**Review by piggggggggy — APPROVED** [review]
LGTM~

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue (hunk 1 above)
Comment (sulmoJ) [reply packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:63]: Oh...

File: apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue (hunk 2 above)
Comment (sulmoJ) [reply apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue:42]: Sample!

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue (hunk 1 above)
Comment (skdud4659) [reply packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:63]: ㅋㅋㅋㅋㅋ

**Pull request merged**